examples/mnist/export_test.py

```diff
@@ -0,0 +1,64 @@
+import tensorflow as tf
+import tensorflow_hub as hub
+import unittest
+import tempfile
+import shutil
+import os
+from absl.testing import absltest
+import export
+from export import train_and_export
+
+TMPDIR = tempfile.mkdtemp()
+model = None
+TEST_DATA_DIR = "examples/mnist/test_data"
+
+def load_model(path):
+  global model
+  if not model:
+    model = hub.load(path)
+  return model
+
+
+class TFHubMNISTTest(tf.test.TestCase):
+  def setUp(self):
+    file_ = tf.gather(tf.io.matching_files("%s/*.jpg" % TEST_DATA_DIR), 0)
+    self.test_image = tf.expand_dims(
+        tf.image.decode_jpeg(
+            tf.io.read_file(file_)), 0)
+    self.test_label = int(str(file_.numpy()).split("/")[-1].split(".jpg")[0])
+
+  def test_model_exporting(self):
+    mocked_dict = export.sys.__dict__.copy()
```

Comment:
What's going on here? Not sure what this test is doing.

Reply:
This test is training the model for one iteration and exporting the graph. The reason for this test is to check if the trained model is being exported properly or not.

Reply:
I meant the mocking of sys.__dict__. I have a strong feeling that this is not needed.